### PR TITLE
Add github action to deploy patsy (the proxy)

### LINKED ISSUE #1667: Configure and deploy payment proxy to digital ocean
Deploy the payment proxy to digital ocean, using the hardcoded payee,payer,hub cloud nodes,

### PR DESCRIPTION
**Pull request by geoknee**
Closes #1667

### CODE REVIEW

File: .github/workflows/deploy-patsy.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  deploy-patsy:
+    needs: build-image
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Deploy to Droplet
+        env:
+          DO_API_KEY: ${{ secrets.DO_API_KEY }}}
+          DROPLET_IP: "64.227.75.247"
```
Comment (geoknee) [inline .github/workflows/deploy-patsy.yml:31]: I made a new droplet on DO for this.

File: .github/workflows/deploy-patsy.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: Deploy Cloud Nodes
```
Comment (geoknee) [inline .github/workflows/deploy-patsy.yml:1]:
```suggestion
name: Deploy Payment Proxy Server
```

File: .github/workflows/deploy-patsy.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          NAME: nitro-payment-proxy
+        run: |
+          echo "$SSH_PRIVATE_KEY" > private_key.pem
+          chmod 600 private_key.pem
+          ssh -o StrictHostKeyChecking=no -i private_key.pem root@$DROPLET_IP <<ENDSSH
+            docker login -u $DO_API_KEY -p $DO_API_KEY registry.digitalocean.com
+            docker pull registry.digitalocean.com/magmo/nitro-payment-proxy:latest
+            docker stop $NAME || true
+            docker rm $NAME || true
+            docker run --restart=unless-stopped -it -d --name $NAME -p 443:443 -p 80:80 -p 5511:5511 -e registry.digitalocean.com/magmo/nitro-payment-proxy:latest
```
Comment (bitwiseguy) [inline .github/workflows/deploy-patsy.yml:41]: There is a dangling `-e` on this line

File: Makefile [hunk 4]
```diff
@@ -17,6 +17,10 @@ docker/local/start:
 
 docker/paymentproxy/build:
 	docker build -f docker/paymentproxy/Dockerfile -t payment-proxy .
```
Comment (bitwiseguy) [inline Makefile:19]: There is a mismatch in names between this command and the `push` command below. Here you tag it as `payment-proxy` but below you reference it as `nitro-payment-proxy`

**Review by lalexgap — APPROVED** [review]

**Review by bitwiseguy — APPROVED** [review]

**Pull request merged**